app/tests/unit/services/version.spec.js

```diff
@@ -293,3 +296,57 @@ describe('updateIsLatest', () => {
     expect(versionTrx.commit).toHaveBeenCalledTimes(1);
   });
 });
+
+describe('RemoveDuplicateLatest', () => {
+  it('sets all other versions to isLatest=false', async () => {
+    listAllObjectVersionsSpy.mockResolvedValue({
+      DeleteMarkers: [{}],
+      Versions: [{ IsLatest: true, VersionId: validUuidv4 }]
+    });
+    objectSpy.mockResolvedValue({
+      path: '/test',
+      bucketId: '0000-0000'
+    });
+    Version.where.mockResolvedValueOnce([{ isLatest: true },{ isLatest: true }]);
```

Comment:
Style: Double check and make sure you're right clicking and formatting with VSCode - spacing in the object array seems a bit off.